index.html

```diff
@@ -0,0 +1,130 @@
+</head>
+<body>
+    <header></header>
+    <nav>
+        <div class="mobile-menu-open">
+            <img src="./icons/hamburger.png" alt="Open menu Options">
+        </div>
+        <div class="mobile-menu-close">
+            <img src="./icons/close.png" alt="Close menu Options">
+        </div>
+        <div class="top-menu">
+            <ul class="top-menu-1">
+                <li><a><img src="./icons/facebook.svg" alt="facebook-icon"></a></li>
+                <li><a><img src="./icons/twitter.svg" alt="twitter-icon"></a></li>
+            </ul>
+            <ul class="top-menu-2">
+                <li><a href="">Language</a></li>
+                <li><a href="">My Account</a></li>
+                <li><a href="">Logout</a></li>
+            </ul>
+        </div>
+        <div class="desktop-menu">
+            <a href="#main-page"><img class="desktop-logo" src="./imgs/backgrounds/AboutUsImg2.png" alt="Space Tourism Conference">
+            </a>
+            <ul class="list-menu">
+                <li class="programme1"><a href="#programme">Program</a></li>
+                <li class="speakers1"><a href="#speakers">Speakers</a></li>
+                <li class="about1"><a href="aboutUs.html">About</a></li>
+                <li class="sponsers1"><a href="#sponsers">Sponsers</a></li>
+                <li class="events1"><a href="aboutUs.html#events">Events</a></li>
+                <li class="register"><a href="https://events.eventzilla.net/e/space-tourism-conference-2022-profiting-from-the-space-experience-economy-2138803014?preview=1626327024469">Register</a></li>
+
+            </ul>
+        </div>
+    </nav>
+    <section class="top">
+        <div class="welcome-content">
+            <h2 class="welcome-content-h2">Welcome to the</h2>
+            <h1 class="welcome-content-h1">Space Tourism Conference
+                <span>April 2022</span>
+            </h1>
+            <div class="paragraph">
+                <p>Space Tourism Conference (STC) presents the cross-section of industries comprising space tourism: aerospace, media, entertainment, architecture, design, advertising, marketing, finance, high tech, science, manufacturing, futurism and the arts. </p>
+            </div>
+            <h3>2022.04.27(WED) ~ 28(THU)</h3>
+            <h4>@ The Space Experience Economy
+            Renaissance Hotel LAX, Los Angeles</h4>
+        </div>
+    </section>
+    <section class="main-programme" id="programme">
+        <div class="main-programme-content">
+            <h3>Event Schedule</h3>
+            <div class="line1"></div>
+            <div class="programme-cards">
+                <div class="programme-card">
+                    <img src="./icons/a.png" alt="Registration & VIP reception">
+                    <h4>Registration & VIP Reception</h4>
+                    <p>Registration at Renaissance Los Angeles Airport Hotel. Timing is from 01:00pm to 03:00pm April 27,2022.</p>
+                </div>
```

Comment:
- [ ] Kindly make the cards wrapper an article to make your code more semantic and readable.
```
<article class="programme-card">
    <img src="./icons/a.png" alt="Registration & VIP reception">
    <h4>Registration & VIP Reception</h4>
    <p>Registration at Renaissance Los Angeles Airport Hotel. Timing is from 01:00pm to 03:00pm April 27,2022.</p>
</article>
```